CC/Sounder/include/BaseRadioSetUHD.h

```diff
@@ -0,0 +1,62 @@
+/**
+ * @file BaseRadioSetUHD.h
+ * @brief Declaration file for the BaseRadioSetUHD class.
+ */
+
+#ifndef BASE_RADIO_SET_UHD_H_
+#define BASE_RADIO_SET_UHD_H_
+
+#include "RadioUHD.h"
+#include "SoapySDR/Formats.hpp"
+#include "SoapySDR/Time.hpp"
```

Comment:
Do you need to include the soapy headers here?  Add necessary system headers for size_t, void, std::atomic.

Reply:
Shouldn't need json included here

Reply:
soapy headers are still necessary, some types like SOAPY_SDR_CS16, I still used those.
I checked with deleting the json include, it gives errors.